assets/scss/components/_toggle.scss

```diff
@@ -0,0 +1,52 @@
+/*
+
+Toggle
+
+Show and hide content base on a user interaction.
+
+Mandatory JS hooks:
+ * `data-target` - element to be shown/hidden
+ * `data-trigger` - class used on the triggering elements i.e. an open button
+ * `data-open` - the `id` of the element used to show the `data-target`
+ * `data-close` - the `id` of the element used to hide the `data-target`
+
+Optional Classes:
+ * `visible-javascript-on` - adding this class to the parent of the `target` ensure it is shown if javascript is turned off.
```

Comment:
> adding this class to the parent of the `target` ensure it is shown if javascript is turned off.

Is wrong, the class `visible-javascript-on` (which has been replaced by `js-visible`) shows an element only when `JavaScript` is on